src/site/content/en/blog/vitals-field-measurement-best-practices/index.md

```diff
@@ -118,11 +145,11 @@ section.
 Report](https://user-images.githubusercontent.com/326742/101584324-3f9a0900-3992-11eb-8f2d-182f302fb67b.png)
 
 {% Aside %}
-  Tip: The [`web-vitals`](https://github.com/GoogleChrome/web-vitals)
-  JavaScript library provides an ID for each metric instance reported making
-  it possible to build distributions in most analytics tools. See the
-  [`Metric`](https://github.com/GoogleChrome/web-vitals#metric) interface
-  documentation for more details.
+Tip: The [`web-vitals`](https://github.com/GoogleChrome/web-vitals)
```

Comment:
Just curious if there's a reason you favor not indenting here.

Reply:
No so much a preference here, just didn't know!

Reply:
Gotcha. I don't have a preference either; I was curious to know if it was a web.dev style thing I was unaware of.